images/macos/provision/assets/select-xamarin-sdk-v2.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e -o pipefail
+
+get_framework_path() {
+  case $1 in
+  --mono) echo '/Library/Frameworks/Mono.framework/Versions' ;;
+  --ios) echo '/Library/Frameworks/Xamarin.iOS.framework/Versions' ;;
+  --android) echo '/Library/Frameworks/Xamarin.Android.framework/Versions' ;;
+  --mac) echo '/Library/Frameworks/Xamarin.Mac.framework/Versions' ;;
+  *) ;;
+  esac
+}
+
+change_framework_version() {
+  local framework=$1
+  local version=$2
+
+  echo "Select $framework $version"
+
+  local countDigit=$(echo "${version}" | grep -o "\." | grep -c "\.")
+  if [[ countDigit -gt 1 ]]; then
+    echo "[WARNING] It is not recommended to specify version in "a.b.c.d" format because your pipeline can be broken suddenly in future. Use "a.b" format."
```

Comment:
I'd suggest getting rid of "a.b.c.d" and rephrase this message a bit.

```suggestion
echo "[WARNING] It is not recommended to specify the exact framework version because your build can be broken with the next patch update. Consider using "major.minor" only format."
```